SU2_CFD/src/variables/CNEMOEulerVariable.cpp

```diff
@@ -117,31 +90,8 @@ CNEMOEulerVariable::CNEMOEulerVariable(su2double val_pressure,
   Gamma.resize(nPoint)           = su2double(0.0);
 
   /*--- Compressible flow, gradients primitive variables ---*/
-  Gradient_Primitive.resize(nPoint,nPrimVarGrad,nDim,0.0);
   Gradient.resize(nPoint,nVar,nDim,0.0);
```

Comment:
And this too?

Reply:
this is used for primitive reconstruction, and for diffusion terms in the NS solver. I believe that the move to CFlowVariable seems ok

Reply:
I am skeptical that "Gradient" is used anymore.  After a quick look, I believe it may be unused.

Reply:
But "Gradient" does not appear to be used.